src/gba/memory.h

```diff
@@ -143,6 +144,8 @@ struct GBAMemory {
 	int32_t eventDiff;
 
 	bool mirroring;
+
+	struct GBAVFameCart vfame;
```

Comment:
Minor nitpick, but this should be put with the other stuff closer to the top, e.g. right under `savedata`.